src/api/block.rs

```diff
@@ -277,7 +280,234 @@ fn tokenomics_transaction_operations(block_summary: &BlockSummary) -> Vec<Operat
                     metadata:             None,
                 })
             }
+            SpecialTransactionOutcome::PaydayPoolReward {
+                pool_owner,
+                ..
+            } => {
+                // The events are ordered such that PaydayPoolReward events are followed
+                // by PaydayAccountReward events for the accounts in the given pool.
+                current_pool_owner = *pool_owner;
+            }
+            SpecialTransactionOutcome::PaydayAccountReward {
+                account,
+                transaction_fees,
+                baker_reward,
+                finalization_reward,
+            } => {
+                if transaction_fees.microccd != 0 {
```

Comment:
Does the API not allow 0 amounts in operations?

Reply:
It does I just didn't think zero rewards were worth including?

Reply:
I agree it is not worth including. I was just wondering whether it is required.